### DISCUSSION TITLE
GitBook mobile app?

### DISCUSSION THREAD

**Discussion by rgunindi** [GitbookIO/community#238] [0. General bugs and questions] [answered]
I didn`t see Gitbook mobile app. 

Don't develop mobile app gitbook like notion mobile app?

**Comment by taszap** [comment, marked as answer]
Hello! 

We do not have a mobile app. 
You can only view GitBook content on mobile (in your browser).

**Reply by rgunindi** [reply]
Are you planning to launch a mobile app?

**Reply by taszap** [reply]
Not at this stage I'm afraid. It isn't really a direction we are pursuing at the moment.

**Comment by GvcZhang** [comment]
Does Gitbook have Desktop App?

**Reply by xaviernogueira** [reply]
Seconding. I am positive I used it before, but I can't find a download link anywhere. @taszap ?
